README.adoc

```diff
@@ -208,20 +237,45 @@ mvn pact:publish
 After the file is published, you'll see a similar output to the following example:
 [role="no_copy"]
 ```
---- maven:4.1.0:publish (default-cli) @ inventory ---
+--- maven:4.1.21:publish (default-cli) @ inventory ---
 Publishing 'Inventory-System.json' with tags 'open-liberty-pact' ... OK
 ```
 
 == Verifying the pact in the Pact Broker
 
-Refresh the Pact Broker webpage at the http://localhost:9292/[^] URL to verify that a new entry exists. There isn't yet a
-timestamp in the last verified column because the pact hasn't been verified by the `system` microservice.
+// Static guide instruction
+ifndef::cloud-hosted[]
+Refresh the Pact Broker webpage at the http://localhost:9292/[^] URL to verify that a new entry exists.
+There isn't yet a timestamp in the last verified column because the pact hasn't been verified
```

Comment:
There isn't yet a timestamp in the last verified column because the pact hasn't been verified by the `system` microservice.

-->

The last verified column doesn't show a timestamp because the `system` microservice hasn't verified the pact yet.